Modules/SubModule_Import/xASL_imp_NII2BIDS_Session.m

```diff
@@ -0,0 +1,71 @@
+function xASL_imp_NII2BIDS_Session(imPar, bidsPar, studyPar, listSessions, nameSubjectSession, bidsLabel, iSession)
+%xASL_imp_NII2BIDS_SubjectSession NII2BIDS conversion for a single sessions.
```

Comment:
a single sessions -> a single session.
Also, add that this is a BIDS session and legacy visit.

Reply:
Ah yes, fixed the other comments in the header too 9c36709 👍